app/models/user_skill.rb

```diff
@@ -2,6 +2,6 @@ class UserSkill < ActiveRecord::Base
   belongs_to :user
   belongs_to :skill
   has_many :endorsements
-  validates_presence_of :user_id
-  validates_presence_of :skill_id
+  validates_presence_of :user
+  validates_presence_of :skill
```

Comment:
Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.